docs/generated/devkit/CreateDependencies.md

```diff
@@ -15,6 +12,9 @@ Use [validateDependency](../../devkit/documents/validateDependency) to validate
 
 ▸ (`options`, `context`): [`RawProjectGraphDependency`](../../devkit/documents/RawProjectGraphDependency)[] \| `Promise`\<[`RawProjectGraphDependency`](../../devkit/documents/RawProjectGraphDependency)[]\>
 
+A function which parses files in the workspace to create dependencies in the [ProjectGraph](../../devkit/documents/ProjectGraph)
```

Comment:
the docs should not be changed. could you run `pnpm documentation` and make sure no docs got updated?

Reply:
as you said, no docs updated after `pnpm documentation`. Should I revert those file or you have a better idea?

Reply:
@xiongemi done